Linking_Charities/database_api.py

```diff
@@ -0,0 +1,21 @@
+from charity.models import Charity
+
+def addCharity(**data):
+  charity = Charity()
+  charity.charity_name = data[charity_name]
```

Comment:
Please change this to charity.name (similar applies to the rest of the fields), it leads to code in the frontend that looks like

```html
<h4>{{charity.charity_name}} - {{charity.charity_description}}</h4>
```

Because the model for the frontend follows that of the backend in order to use efficient type conversions.

```typescript
// This leads to clearer code
export class Hero {
  id:number;
  name:string;
}

// As opposed to this
export class Charity {
  charity_description: string;
  charity_name: string;
  charity_register_id: number;
  id: number;
}
```